tools/targets/ARM_MUSCA_A1/imgtool_lib/keys.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2017 Linaro Limited
+# Copyright (c) 2017-2018, Arm Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Cryptographic key management for imgtool.
+"""
+
+from __future__ import print_function
+from Crypto.Hash import SHA256
+from Crypto.PublicKey import RSA
+from Crypto.Signature import PKCS1_v1_5, PKCS1_PSS
```

Comment:
I see you're using the `Crypto` package here. Can you please add the `pycryptodome` package to the `requirements.txt`? https://github.com/ARMmbed/mbed-os/blob/master/requirements.txt